app/src/main/AndroidManifest.xml

```diff
@@ -81,6 +81,9 @@
       android:name="io.fabric.ApiKey"
       android:value="${fabricApiKey}" />
 
+    <activity
+      android:name="com.connfa.eventdetails.EventDetailsActivity"
+      android:theme="@style/Theme.Connfa.EventDetails" />
```

Comment:
Maybe we can rename the themes something like "Theme.OurAppNameWhichIDontKnow.EventDetails"?